packages/components/src/internal/query/api.ts

```diff
@@ -565,7 +566,26 @@ export function handleSelectRowsResponse(json): any {
     };
 }
 
-export function searchRows(selectRowsConfig, token: any, exactColumn?: string): Promise<ISelectRowsResult> {
+// exported for jest testing
+export function quoteValueColumnWithDelimiters(selectRowsResult: ISelectRowsResult, valueColumn: string, delimiter: string): ISelectRowsResult {
+    const rowMap = selectRowsResult.models[selectRowsResult.key];
+    Object.keys(rowMap).forEach(key => {
+        if (rowMap[key][valueColumn]) {
+            Object.assign(rowMap[key],
+                {
+                    [valueColumn]: {
+                        value: quoteValueWithDelimiters(rowMap[key][valueColumn].value, delimiter),
+                        displayValue: rowMap[key][valueColumn].value,
```

Comment:
should this have been `displayValue: rowMap[key][valueColumn].displayValue,` or was it intentional to use `value` here?